mason/breeders_toolbox/genotyping_trials/upload_genotyping_data_dialogs.mas

```diff
@@ -549,6 +593,277 @@ $facilities
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="ssr_protocol_saved_dialog_message" name="ssr_protocol_saved_dialog_message" tabindex="-1" role="dialog" aria-labelledby="SSRProtocolSavedDialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title" id="SSRProtocolSavedDialog">Success</h4>
+            </div>
+            <div class="modal-body">
+                <div class="container-fluid">
+                    <p>
+                        <span class="ui-icon ui-icon-circle-check" style="float: left; margin: 0 7px 50px 0;"></span>
+                        SSR marker info was saved successfully. You can now proceed with SSR genotyping data upload.
+                    </p>
+                </div>
+            </div>
+            <div class="modal-footer">
+                <button id="dismiss_ssr_protocol_saved_dialog" type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="modal fade" id="upload_ssr_protocol_dialog" name="upload_ssr_protocol_dialog" tabindex="-1" role="dialog" aria-labelledby="uploadSSRProtocolDialog">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title" id="uploadSSRProtocolDialog">Upload SSR Protocol (Marker Info) </h4>
+            </div>
+            <div class="modal-body">
+                <div class="container-fluid">
+                    <form class="form-horizontal" role="form" method="post" enctype="multipart/form-data" encoding="multipart/form-data" id="upload_ssr_protocol_form" name="upload_ssr_protocol_form">
+                        <div class="form-group">
+                            <label class="col-sm-5 control-label">SSR Protocol Name:</label>
+                            <div class="col-sm-7">
+                                <input class="form-control" id="upload_ssr_protocol_name" name="upload_ssr_protocol_name" type="text" placeholder=" e.g. SSR Cassava set 1"/>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label class="col-sm-5 control-label">Species:</label>
+                            <div class="col-sm-7">
+                                <input class="form-control" id="upload_ssr_species_name_input" name="upload_ssr_species_name_input" type="text" placeholder=" e.g. Manihot esculenta"/>
```

Comment:
you can add the autocomplete for this similar to the "upload_genotypes_species_name_input" field

Reply:
@nickmorales OK, thanks Nick. Wii add.